isis/src/base/apps/caminfo/tsts/csv/Makefile

```diff
@@ -7,9 +7,11 @@ commands:
 	$(APPNAME) FROM=$(OUTPUT)/input.cub \
 	TO=$(OUTPUT)/temp.txt FORMAT=flat APPEND=false \
 	STATISTICS=true CAMSTATS=true GEOMETRY=true spice=true > /dev/null;
+	# Remove the trailing decimal place on numbers
+	# Replace the caminfo,version number | date with "caminfo,version | date"
 	cat $(OUTPUT)/temp.txt | \
 	  sed 's/\([0-9]*\.[0-9]\{4\}\)\([0-9]*\)/\1/g' | \
-	  sed 's/\([a-z]*\,\)\([0-9]\.[0-9]*\.[0-9]*\.[A-Za-z0-9]* [a-z]* | [0-9]\{4\}\-[0-9]\{2\}\-[0-9]\{2\}\)/\1/g' \
+	  sed 's/\([a-z]*\,\)\([0-9]\.[0-9]*\.[0-9]*.*| [0-9]\{4\}\-[0-9]\{2\}\-[0-9]\{2\}\)/\1version | date/g' \
```

Comment:
I was just looking into this as well. I pretty much reached the same solution, although I wasn't sure what the original sed command wasn't doing, but if it was unimportant then this is a good fix.